cardano-config/src/Cardano/Tracing/ToObjectOrphans.hs

```diff
@@ -961,3 +983,205 @@ instance (Show (GenTx blk), Show (GenTxId blk))
         , "txsNoLongerValidRemoved" .= String (pack $ show $ txs1)
         , "mempoolSize" .= String (pack $ show $ mpSz)
         ]
+
+instance ( StandardHash blk
+         , ToObject (LedgerError blk)
+         , ToObject (ValidationErr (BlockProtocol blk)))
+      => ToObject (ChainDB.InvalidBlockReason blk) where
+  toObject verb (ChainDB.ValidationError extvalerr) =
+    mkObject
+        [ "kind"  .= String "ValidationError"
+        , "error" .= toObject verb extvalerr
+        ]
+  toObject verb (ChainDB.InChainAfterInvalidBlock point extvalerr) =
+    mkObject
+        [ "kind"  .= String "InChainAfterInvalidBlock"
+        , "point" .= String (pack $ show point)
+        , "error" .= toObject verb extvalerr
+        ]
+
+instance ( StandardHash blk
+         , ToObject (LedgerError blk)
+         , ToObject (ValidationErr (BlockProtocol blk))
+         )
+      => ToObject (ExtValidationError blk) where
+  toObject verb (ExtValidationErrorLedger err) = toObject verb err
+  toObject verb (ExtValidationErrorHeader err) = toObject verb err
+
+instance ( StandardHash blk
+         , ToObject (ValidationErr (BlockProtocol blk)))
+      => ToObject (HeaderError blk) where
+  toObject verb (HeaderProtocolError err) =
+    mkObject
+        [ "kind"  .= String "HeaderProtocolError"
+        , "error" .= toObject verb err
+        ]
+  toObject verb (HeaderEnvelopeError err) =
+    mkObject
+        [ "kind"  .= String "HeaderEnvelopeError"
+        , "error" .= toObject verb err
+        ]
+
+instance (StandardHash blk)
+ => ToObject (HeaderEnvelopeError blk) where
+  toObject _verb (UnexpectedBlockNo expect act) =
+    mkObject
+        [ "kind"  .= String "UnexpectedBlockNo"
+        , "expected" .= condense expect
+        , "actual"   .= condense act
+        ]
+  toObject _verb (UnexpectedSlotNo expect act) =
+    mkObject
+        [ "kind"  .= String "UnexpectedSlotNo"
+        , "expected" .= condense expect
+        , "actual"   .= condense act
+        ]
+  toObject _verb (UnexpectedPrevHash expect act) =
+    mkObject
+        [ "kind"  .= String "UnexpectedPrevHash"
```

Comment:
In `STYLE.md`: * Do not vertically align code. Mainly because the need to change identifiers and re-aligning the other lines causes semantically null changes in the diff.